### ISSUE TITLE
Perceptual loss weight 

### ISSUE THREAD

**Issue by YuraYelisieiev** [saic-mdal/lama#88]
In your paper, you write that:
**Naive supervised losses require the generator to reconstruct
the ground truth precisely. However, the visible parts of the
image often do not contain enough information for the exact
reconstruction of the masked part. Therefore, using naive
supervision leads to blurry results due to the averaging of
multiple plausible modes of the inpainted content. 
In contrast, perceptual loss evaluates a distance between features extracted from the predicted and the target
images by a base pre-trained network**

But inside some main configs, you set the perceptual weight to 0.
Is it a config problem, or do you train the models without perceptual loss?

```
perceptual:
    weight: 0
```
In `lama-fourier` config

**Comment by windj007** [comment]
`losses.perceptual.weight` corresponds to VGG-based perceptual loss, which are actually do not used in our best-performing models

However, note another `losses.resnet_pl.weight=30`, which corresponds to Segmentation-based PL.

**Comment by YuraYelisieiev** [comment]
I see, thanks for the answer
